integration_tests/src/main/python/window_function_test.py

```diff
@@ -415,7 +415,23 @@ def test_window_aggs_for_ranges_timestamps(data_gen):
         'from window_agg_table',
         conf = {'spark.rapids.sql.castFloatToDecimal.enabled': True})
 
-_gen_data_for_collect = [
+@pytest.mark.xfail(reason="[UNSUPPORTED] Ranges over non-timestamp columns "
+                          "(https://github.com/NVIDIA/spark-rapids/issues/216)")
```

Comment:
This was closed. Is it still an issue?

Reply:
Sorry, I mistakenly kept this test by when resolving merge conflicts.